simplesat/sat/policy.py

```diff
@@ -82,12 +140,16 @@ def _group_packages_by_name(self, decision_set):
 
     def _handle_empty_decision_set(self, assignments, clauses):
         # TODO inefficient and verbose
+
+        # The assignments with value None
```

Comment:
These comments are here because it took me quite a while to understand what this policy was doing. For someone who knows the code base, they do not add much value. I'll remove them now.